.github/workflows/build.yml

```diff
@@ -86,6 +86,9 @@ jobs:
           - { cxx: g++,     image: '2022-clang11',   abi: '9',  build: 'Release', cmake: '-DDISABLE_DEPENDENCY_VERSION_CHECKS=ON' }
       fail-fast: false
     steps:
+    - name: Enable Node 16
+      run: |
+        echo "ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION=true" >> $GITHUB_ENV
```

Comment:
Thanks @apradhana, this seems to fix the nodejs-related CI issues which appears to be causing wide-spread chaos with GHA, not just for VDB. However, I think this is only an issue for the old vfx 2022 images though, it would be good if we can gate this temporary fix to just those versions so that when we remove the 2022 images, it's more obvious we can also remove this temporary fix. I think we only have to support older glibc for a few more months (until VDB 12.0 is released).

I haven't tried this, but something like `if: contains(matrix.config.image, '2022') == true` if that works?